packages/ocplib-json-typed-bson/ocplib-json-typed-bson.0.7/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+synopsis: "A Json_repr compatible implementation of the JSON compatible subset of BSON"
+maintainer: "Benjamin Canou <benjamin@ocamlpro.com>"
+authors: "Benjamin Canou <benjamin@ocamlpro.com>"
+homepage: "https://github.com/ocamlpro/ocplib-json-typed"
+bug-reports: "https://github.com/ocamlpro/ocplib-json-typed/issues"
+license: "LGPLv3 w/ linking exception"
+dev-repo: "git+https://github.com/ocamlpro/ocplib-json-typed.git"
+
+build: [ "dune" "build" "-j" jobs "-p" name "@install" ]
+run-test: [ "dune" "runtest" "-p" name "-j" jobs ]
```

Comment:
this should be in the `build` stanza but annotated with `{with-test}` now

Reply:
mmh, both does the same thing as far as I am aware

Reply:
then why do we have both... :-)

Reply:
I think the proper way is using `run-test` which runs the commands during the `test phase`. If you annotate with `{with-test}` it's done during the `build phase`. I think the annotation is primarily here to enable tests flags or something like that during configuration or compilation.

I'm not sure what's the difference between the `build phase` and the `test phase` (maybe there can be some differences from the bubblewrap point of view) but they are morally the same in the end, just one usage is more a distorted use from its original use I suppose.

cc @AltGr @rjbou